extension-style/src/main/java/com/mapbox/maps/extension/style/atmosphere/generated/AtmosphereExt.kt

```diff
@@ -24,4 +25,11 @@ fun StyleInterface.setAtmosphere(atmosphere: StyleContract.StyleAtmosphereExtens
   atmosphere.bindTo(this)
 }
 
+/**
+ * Removes atmosphere from style if it was set.
+ */
+fun StyleInterface.removeAtmosphere() {
+  setStyleAtmosphere(Value.nullValue())
+}
```

Comment:
btw, should we have `removeLight` as well?
We do have `removeAtmosphere` and `removeTerrain` now

Reply:
Seems it's introduced but then removed in https://github.com/mapbox/mapbox-maps-android/pull/974, I guess it doesn't really work